invokeai/app/services/model_install/model_install_default.py

```diff
@@ -559,11 +556,11 @@ def _sync_model_path(self, key: str) -> AnyModelConfig:
         return model
 
     def _scan_register(self, model: Path) -> bool:
-        if model in self._cached_model_paths:
+        if model.resolve() in self._cached_model_paths:
```

Comment:
Is this the change that fixes symbolic links?

Reply:
Yes.